prereise/gather/griddata/hifld/data_process/generators.py

```diff
@@ -0,0 +1,71 @@
+import pandas as pd
+
+from prereise.gather.griddata.hifld.data_access import load
+
+
+def map_generator_to_bus_by_location(generator, substation_groupby):
```

Comment:
Should it be: `map_generator_to_sub_by_location`?

Reply:
Eventually we need to have the generators mapped to bus IDs. This function will probably need another input (e.g. a bus table) to do that properly.

Reply:
Fulfilling #205 should give us bus IDs, and then we probably want to connect each generator to the lowest-voltage bus within each substation.